modules/gapi/src/backends/fluid/gfluidcore_func.simd.hpp

```diff
@@ -1973,14 +1956,448 @@ ABSDIFFC_SIMD(float)
+}
+
+#define DIVRC_SIMD(SRC, DST)                                                   \
+int divrc_simd(const float scalar[], const SRC in[], DST out[],                \
+               const int length, const int chan, const float scale)            \
+{                                                                              \
+    switch (chan)                                                              \
+    {                                                                          \
+    case 1:                                                                    \
+    case 2:                                                                    \
+    case 4:                                                                    \
+    {                                                                          \
+        if (std::fabs(scale - 1.0f) <= FLT_EPSILON)                            \
+        {                                                                      \
+            return divrc_simd_common(not_scale_tag{}, in, scalar,              \
+                                     out, length, scale);                      \
+        }                                                                      \
+        else                                                                   \
+        {                                                                      \
+            return divrc_simd_common(scale_tag{}, in, scalar, out,             \
+                                     length, scale);                           \
+        }                                                                      \
+    }                                                                          \
+    case 3:                                                                    \
+    {                                                                          \
+       if (std::fabs(scale - 1.0f) <= FLT_EPSILON)                             \
+        {                                                                      \
+            return divrc_simd_c3(not_scale_tag{}, in, scalar,                  \
+                                 out, length, scale);                          \
+        }                                                                      \
+        else                                                                   \
+        {                                                                      \
+            return divrc_simd_c3(scale_tag{}, in, scalar, out,                 \
+                                 length, scale);                               \
+        }                                                                      \
+    }                                                                          \
+    default:                                                                   \
+        GAPI_Assert(chan <= 4);                                                \
```

Comment:
i think there may be warning for walking through switch-cases

Reply:
There is no warnings.